src/github.com/getlantern/byteexec/byteexec_test.go

```diff
@@ -47,7 +48,7 @@ func TestExec(t *testing.T) {
 	}
 	be = createByteExec(t, data)
 	updatedInfo = testByteExec(t, be)
-	assert.Equal(t, fileMode, updatedInfo.Mode(), "File mode is changed back to %v", fileMode)
+	assert.True(t, fileMode == updatedInfo.Mode(), fmt.Sprintf("File mode was %v instead of %v", updatedInfo.Mode(), fileMode))
```

Comment:
Not that it matters, since the test is essentially the same, but did you replace assert.Equal w/ assert.True for a particular reason related to the upgrade?